recipes/7bitdi/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
I think we are defaulting to `3.15` as the minimum version now.

```suggestion
cmake_minimum_required(VERSION 3.15)
```

Reply:
Changed version to 3.15